utils/message-generator/src/main.rs

```diff
@@ -179,9 +180,22 @@ pub struct SaveField {
     keyword: String,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+pub enum Condition {
+    WaitUntil,
+}
```

Comment:
```suggestion
// enum as a placeholder for potential conditions to be implemented in the future
#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
pub enum Condition {
    WaitUntil,
}
```

Reply:
@lorbax do you agree with this suggestion?

if yes, you can simply clicl the `Commit suggestion` button above, no need to manually implement this on your branch

Reply:
> Do you have a particular reason for adding a commit with this small suggestion?

not really! feel free to force push from your local changes